partcad/src/partcad/shape.py

```
@@ -457,15 +460,54 @@ async def render_png_async(
                     height = png_opts["height"]
                 else:
                     height = DEFAULT_RENDER_HEIGHT
+            if line_weight is None:
+                if "lineWeight" in png_opts and not png_opts["lineWeight"] is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)